pay/example/lib/main.dart

```diff
@@ -124,6 +127,75 @@ class _PaySampleAppState extends State<PaySampleApp> {
               fontSize: 15,
             ),
           ),
+          const SizedBox(height: 15),
+          Row(
```

Comment:
I suggest we keep the sample screen as simple as possible to make it easier to navigate.
We can use logic like this to verify that the button reacts to price changes, and leave it out of the final merge.